### PR TITLE
Fact sheet

### PR DESCRIPTION
**Pull request by strbean**
Adds Facts and Stats page with lots of static content and live totals for variants.

Also remove old links from navbar, remove CIMB logo from front page, and add ETH Zurich and UCSC Genomics Institute logos.

### CODE REVIEW

File: website/js/FactSheet.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+var {Link} = require('react-router');
+
+var FactSheet = React.createClass({
+    mixins: [PureRenderMixin],
+    getInitialState: function() {
+        return {};
+    },
+    componentWillMount: function() {
+        backend.variantCounts().subscribe(
+            resp => this.setState(this.setState(resp)),
```
Comment (zfisch) [inline website/js/FactSheet.js]: never seen this before (setState inside a setState call), is this intentional and necessary?

File: website/js/FactSheet.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+            () => this.setState({error: 'Problem connecting to server'}));
+    },
+    render: function () {
+        //var {data, page, totalPages, error} = this.state;
```
Comment (zfisch) [inline website/js/FactSheet.js]: pls remove comment if not needed

**Comment by strbean** [comment]
I'll fix all those issues right away! No idea how I managed to put that
nested setState, I'm surprised it didn't blow up when I tested it ><

**Comment by strbean** [comment]
Alright, I fixed those issues. I removed images that were made redundant by logo changes, but I didn't do a thorough search to see if any of the older images were already unused.

**Pull request merged**